modules/fedimint-mint-client/src/oob.rs

```diff
@@ -25,9 +38,11 @@ use crate::{MintClientContext, MintClientStateMachines, SpendableNote};
 /// ```
 #[derive(Debug, Clone, Eq, PartialEq, Hash, Decodable, Encodable)]
 pub enum MintOOBStates {
+    // Obsoleted, like [`Self::CratedMulti`] but for a single note only.
+    Created(MintOOBStatesCreated),
     /// The e-cash has been taken out of the wallet and we are waiting for the
     /// recipient to reissue it or the user to trigger a refund.
-    Created(MintOOBStatesCreated),
+    CreatedMulti(MintOOBStatesCreatedMulti),
```

Comment:
Why keep the old one around? If you add the state in the middle instead of the end you have to write a SM migration anyway and could change all `Created` states into `CreatedMulti`. Alternatively you could add the new variant at the end and we could add an empty SM migration, just to increment the DB version and detect going backwards in versions.

Reply:
I might get rid of them. I haven't considered all the downstream consequences yet, especially w.r.t. notifications etc.

Reply:
If you choose to keep them around I'd advise to add the new variants to the end of the enum to avoid requiring migrations.